src/components/ModalPix.jsx

```diff
@@ -103,32 +100,63 @@ const ModalPix = ({ qrCode, status, mensagemPix, onOpen, onClose }) => {
           </Group>
           <div style={{ alignItems: "center" }}>
-            <Input
-              ref={inputRef}
-              type="text"
-              value={qrCode}
-              readOnly
-              style={{ flex: 1, }}
-            />
-            <Button
-              mt={4}
-              onClick={copyToClipboard}
-              style={{
-                color: icon === "FaCopy" ? "white" : "white",
-                backgroundColor: icon === "FaCopy" ? "" : "green",
-                padding: "8px",
-                cursor: "pointer",
-              }}
-              fullWidth
-            >
-              <span className="material-icons"
-              style={{color: icon === "FaCopy" ? "white" : "white", 
-            }}
-              >
-                {icon === "FaCopy" ? 'Clique para copiar o código': 'Código copiado'} {" "} 
-                {icon === "FaCopy" ? <FaCopy /> : <FaCheck  color="white"/>}
-              </span>
-            </Button>
+            {perfil === "cliente" ? (
+              <div>
+                <Input
+                  ref={inputRef}
+                  type="text"
+                  value={qrCode}
+                  readOnly
+                  style={{ flex: 1 }}
+                />
+                <VoltarComponente space={true} />
+                <Button
+                  mt={4}
+                  onClick={copyToClipboard}
+                  style={{
+                    color: "white",
+                    backgroundColor: icon === "FaCopy" ? "" : "green",
+                    padding: "8px",
+                    cursor: "pointer",
+                  }}
+                  fullWidth
+                >
+                  <span className="material-icons" style={{ color: "white" }}>
+                    {icon === "FaCopy"
+                      ? "Clique para copiar o código"
+                      : "Código copiado"}
+                    {icon === "FaCopy" ? <FaCopy /> : <FaCheck color="white" />}
+                  </span>
+                </Button>
+              </div>
+            ) : (
+              <>
+                <Input
+                  ref={inputRef}
+                  type="text"
+                  value={qrCode}
+                  readOnly
+                  style={{ flex: 1 }}
+                />
+                <div className="pt-4 mb-4 gap-4 d-flex alignItems-center">
+                  <Button
+                    mt={4}
+                    onClick={funcao}
+                    loaderPosition="right"
+                    className="bg-blue-50"
+                    radius="md"
+                    style={{
+                      width: "300px",
+                      height: "50px",
+                      marginBottom: "2px",
+                    }}
+                  >
+                    2ª Via
+                  </Button>
+                  <VoltarComponente fechar={true} />
+                </div>
+              </>
+            )}
```

Comment:
Arrumar aqui, se o perfil for = a monitor exibe os botoes, caso contrario nao

Reply:
Muda ao nome do botao, nao poem segunda via, poem algo tipo ˜imprimir˜ e nao sei se tem a necessidade do componente voltar ali